### DISCUSSION TITLE
How do you talk to marketing?

### DISCUSSION THREAD

**Discussion by justinabrahms** [todogroup/ospology#140] [OSPO Discussions]
Our marketing folks aren't yet super involved in our OSPO. I'd like to get them on board but they have their own agenda and language. In particular, our open source stuff isn't going to drive traffic to our main thing (e.g. people aren't going to buy sneakers bc we make rad open source).

If you're in a similar situation, how do you achieve the marketing functions of raising awareness and getting the message out for your open source code?

**Comment by xmulligan** [comment]
What about from the hiring/employer Branding perspective?

**Comment by anajsana** [comment]
hey, @justinabrahms thanks for bringing this discussion here! As a person whose initial background was in marketing and who later studied and specialized in data science and data engineering, I can give my thoughts as an individual who is familiar with both frameworks and work culture.

IMHO, The main issue (maybe you already probably know) is how marketing people are taught at most of the universities. Somehow, the way of working you get from there is very competitive, and that doesn't fit in an open source culture at all 🥲

And most of the time is not something that marketing people want, is just that is the only way that they were trained to do things! So, here are the two pieces of advice I find useful to improve communication if marketing people are engaging with OSPOs and open source:


1) **Build hybrids:** Have mentorship programs dedicated to marketing to let them understand open source, software development environments (even some basic programming syntax in a specific language to allow them to build things) and encourage them to take courses and build pet projects. Letting them have practical experience of how is it like to work in engineering environments, can build empathy on both sides and will better understand the needs of the other team.

2) **Seek ambassadors who talk the two languages** (for marketing and engineering people) to fill the communication gap between tech and non-tech people.

Fun story, that is what [Bitergia](https://bitergia.com/) folks taught me when I worked there: They open the doors to Open Source, Software development analytics, and Python. Thanks to their mentorship, I was able to advance and study my Master in Data Science and build my Thesis, have a better understanding of open source and its culture, and build empathy with the various teams.

Since open source means everything to me, in my spare time, (and as a way to give back to the community) I do as I can to advocate for the need for marketing people to learn the basics of programming, open source, and how the ecosystem work (I honestly would love to see more advocates doing this to build the bridge 🙂). 

Marketing people might not need to have a full transition to a tech role as I did in the past, but is important to make them participants and not just spectators, IMHO.

**Comment by nicorikken** [comment]
In working with my marketing colleagues I focus on the goal I want to achieve for the open source project. Audience, message and channels will follow logically from the goal and context. That can be quite different than the main corporate marketing.

**Comment by debbryant** [comment]
The OSPO I led for 8 years had its own marketing resources (brand, infrastructure, social media, design etc).  Even though our community goals were well aligned with business goals, the marketing efforts simply had different measurements.  So requiring badge swipes to confirm Butts in Seats at Events (FOSDEM anyone :)? or collecting email addresses for newsletters were simply mismatches in culture and success indicators.

We built modest capabilities (named Outreach, not marketing) which supports a cluster of communities of interest + the broader eocystem, spent time understand Marketing’s goals and places where were could work together to share resource, platform etc.  (i.e think global events, distribution of materials, and so forth).  Community data science was added about four years ago and lifts all corporate boats.

FWIW

Deb Bryant
